scripts/build.py

```diff
@@ -1486,7 +1486,9 @@ def build_docs(self):
         skip_tests = '--skip-tests' if self.skip_tests or self.target_platform != self.host else ''
         self._log('Building API docs')
         cwd = join(self.defold_root, 'engine/docs')
-        cmd = '%s %s/ext/bin/waf configure --prefix=%s %s distclean configure build install' % (' '.join(self.get_python()), self.dynamo_home, self.dynamo_home, skip_tests)
+        python_cmd = ' '.join(self.get_python())
+        cmd = '%s %s/ext/bin/waf configure --prefix=%s %s distclean configure build install' % (python_cmd, self.dynamo_home, self.dynamo_home, skip_tests)
+        run.env_command(self._form_env(), [python_cmd, './scripts/bundle.py', 'docs', '--docs-dir', '%s/share/doc' % self.dynamo_home], cwd = join(self.defold_root, 'editor'))
```

Comment:
Now there will be editor docs in the reference!

Reply:
Can we do it here, though? I thought we relied on the output of this for the `.sdoc` files in `editor/resources/docs` that end up in the editor. But perhaps we don't need a fully formed editor jar to generate the editor script docs?

Reply:
Hmm yeah, we will end up with an outdated `editor_doc.sdoc` in the editor. I'll make sure to exclude it! Yes, we don't need the editor's `editor_doc.sdoc` in the editor, since the editor code is a source of truth for the editor doc.